common/src/main/java/com/microsoft/identity/common/internal/cache/AbstractAccountCredentialCache.java

```diff
@@ -235,11 +236,11 @@ static boolean targetsIntersect(@NonNull final String targetToMatch,
 
         // Add the array values to these sets, lowercasing them
         for (final String target : targetToMatchArray) {
-            soughtTargetSet.add(target.toLowerCase());
+            soughtTargetSet.add(target.toLowerCase(Locale.ROOT));
```

Comment:
@iambmelt - Am a bit concerned about using ROOT here instead of US which appears to be what we did for the rest of caching.  What are your thoughts?

Reply:
https://cs.android.com/android/platform/superproject/+/master:libcore/libart/src/main/java/java/lang/CaseMapper.java;drc=master;l=45 it looks like there are a few cases where the Locale matters - Turkish, Lithuanian, and Azerbaijani, etc.  Other than those and a few special cases it shouldn't matter.